tests/integration/netdev.bats

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+	requires root
+	setup_busybox
+
+	# skip to setup resources that require root
+	# to avoid the test to fail
+	if [ $EUID -ne 0 ] || in_userns; then
+		# create a dummy interface to move to the container
+		ip link add dummy0 type dummy
+
+		# create a temporary name for the test network namespace
+		tmp=$(mktemp)
+		rm -f "$tmp"
```

Comment:
nit: use --dry-run for this. I know you c&p, but I don't think there is a reason to not use -d and avoid the rm later